api/src/convert/tx_in.rs

```diff
@@ -72,6 +72,19 @@ impl From<&InputRules> for external::InputRules {
 
         input_rules.set_max_tombstone_block(source.max_tombstone_block);
 
+        let fractional_outputs = source
+            .fractional_outputs
+            .iter()
+            .map(external::RevealedTxOut::from)
+            .collect();
+        input_rules.set_fractional_outputs(fractional_outputs);
+
+        if let Some(fractional_change) = source.fractional_change.as_ref() {
+            input_rules.set_fractional_change(external::RevealedTxOut::from(fractional_change));
```

Comment:
Optional/Equivalent: Use `into()`
```suggestion
            input_rules.set_fractional_change(fractional_change.into());
```